robot/robot.py

```diff
@@ -53,6 +55,7 @@ def __init__(
         configure_logging()
 
         LOGGER.info("Initializing Hardware...")
+        LOGGER.info("Robot Intiailised (v{})".format(__VERSION__))
```

Comment:
I suggest either we move this down the initialization process, so the robot has actually initialized when we say it is. Or we don't say it's initialised, and move printing the version right to the top of the initialization. I'm fond of the latter

Reply:
fixed in 4b51ef0cb9879fa86d101334bfd7478b7cc02459 also the british english spelling is 'Initialise' :wink: